core/templates/dev/head/services/PromoBarService.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Rename this file to `ResourceService.js`

Reply:
Done.